src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall() {
```

Comment:
Lottery#getRandomBall should return `Ball` not a `String`.

Reply:
why do I need the getRandomBall() method at all? If anyway it will return the created Ball